libs/langchain-community/src/embeddings/alibaba_embedding.ts

```diff
@@ -0,0 +1,215 @@
+import { getEnvironmentVariable } from "@langchain/core/utils/env"
+import { Embeddings, type EmbeddingsParams } from "@langchain/core/embeddings"
+import { chunkArray } from "@langchain/core/utils/chunk_array"
+
+export interface AlibabaAIEmbeddingsParams extends EmbeddingsParams {
```

Comment:
Rename to `AlibabaTongyiEmbeddingsParams` to match naming precedent

Reply:
I changed it